### ISSUE TITLE
[sklearn#34652] MNT Python 3.15 support

### ISSUE THREAD

**Issue by godofecht** [flooooooooooow/flow#299]
**Mirrored from [scikit-learn#34652](https://github.com/scikit-learn/scikit-learn/issues/34652)**

Original title: MNT Python 3.15 support
Original labels: Needs Investigation
Created: 2026-08-05T06:43:06Z

---

> [!WARNING]
> This issue is not yet ready for a PR. If you are interested in contributing to scikit-learn, please have a look at our [contributing guidelines](https://scikit-learn.org/dev/developers/contributing.html), and in particular the sections for [new contributors](https://scikit-learn.org/dev/developers/contributing.html#new-contributors) and the ["Needs triage"](https://scikit-learn.org/dev/developers/contributing.html#issues-tagged-needs-triage) label.

Python 3.15 first rc was released, see [release schedule](https://peps.python.org/pep-0790/#schedule). 
- [x] numpy already has [Python 3.15 dev wheels](https://anaconda.org/channels/scientific-python-nightly-wheels/packages/numpy/files?file_q=315,)
- [ ] but not scipy see https://github.com/scipy/scipy/issues/25530
- [ ] once scipy has development wheels we can start the testing I think

Quick testing in loky seems to reveal that we may have some issues related to the resource manager but maybe that is not a problem for scikit-learn tests. To be investigated.

---

## Flow context

This issue is mirrored from scikit-learn to explore whether Flow encounters the same problem or can approach it differently. Add analysis below.

**Comment by godofecht** [comment]
Moved to https://github.com/godofecht/flow-scikit/issues/124 in the flow-scikit repository. Further library-specific triage and implementation will happen there.
